dao/src/test/java/com/iluwatar/dao/CustomerDaoImplTest.java

```diff
@@ -0,0 +1,81 @@
+package com.iluwatar.dao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class CustomerDaoImplTest {
+
+	private CustomerDaoImpl impl;
```

Comment:
Shouldn't we call `CustomerDaoImpl` as `InMemoryCustomerDao`. It represents what the class does.
@iluwatar Do you think we should do that? We can add a `DatabaseCustomerDao` to it, if not now in future.